frontend/src/components/projects/ProjectsFilters.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import {connect} from "react-redux";
+import {filter as filterObject} from "rambda";
+import {Collapse} from "antd";
+
+import {setFilter} from "../../modules/projects/actions";
+import FilterGroup from "../filters/FilterGroup";
+import {PROJECT_FILTERS} from "../filters/descriptions";
+
+const mapStateToProps = state => ({
+  filters: state.projects.filters,
+});
+
+const actionCreators = {setFilter};
+
+const ProjectsFilters = ({
```

Comment:
No need for detached filters for Projects for now. There are detached filters for Samples because there were some fields that we wanted to have filters on, but these fields were not part of any of the list columns. In the case of Projects, all those filters can be applied to the table columns directly :)